app/components/bali/calendar/component.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="calendar-component">
+  <div class="box <%= period == :month ? 'month-view' : 'week-view' %>">
+    <%= header %>
+
+    <div class="table-container">
+      <table class="table is-fullwidth">
+        <thead>
+          <tr>
+            <% date_range.slice(0, all_week ? 7 : 5).each do |day| %>
+              <th class="has-text-centered has-text-grey-dark has-text-weight-normal is-size-4">
+                <%= t('date.day_names')[day.wday] %>
+              </th>
+            <% end %>
+          </tr>
+        </thead>
+
+        <tbody>
+          <% date_range.each_slice(7) do |week| %>
+            <%= content_tag :tr, class: tr_classes_for(week) do %>
+              <% filter_week(week).each do |day| %>
+                <%= content_tag :td, class: td_classes_for(day) do %>
+                  <% if show_date %>
+                    <% if period == :month %>
+                      <div class="has-text-weight-semibold is-pulled-right is-size-5">
+                        <%= day.day %>
+                      </div>
+                    <% else %>
+                      <h3 class="title is-3 <%= @options[:weekly_title_class] %>"><%= day.day %></h3>
+                    <% end %>
+                  <% end %>
+                  <% if template.present? %>
+                    <%= render template, events: sorted_events[day], period: period, day: day %>
+                  <% end %>
+                <% end %>
+              <% end %>
+            <% end %>
+          <% end %>
+        </tbody>
+      </table>
+    </div>
+
+    <% if footer? %>
+      <%= footer %>
+    <% end %>
```

Comment:
```suggestion
    <% footer  if footer? %>
```
Just clean code.

Reply:
Please check if you even need the **if** statement. I think that if the user didn't define a footer it will just print nothing.